src/main/kotlin/example/Header.kt

```diff
@@ -4,22 +4,22 @@ import csstype.*
 import example.Sizes.Header
 import kotlinext.js.jso
 import kotlinx.browser.window
+import mui.icons.material.Brightness4
+import mui.icons.material.Brightness7
 import mui.icons.material.GitHub
 import mui.icons.material.Menu
 import mui.icons.material.MenuBook
 import mui.material.*
 import mui.material.Size
-import react.FC
-import react.Props
-import react.ReactNode
+import react.*
 import react.dom.aria.AriaHasPopup.`false`
 import react.dom.aria.ariaHasPopup
 import react.dom.aria.ariaLabel
 import react.dom.html.ReactHTML
 import react.router.useLocation
-import react.useContext
 
 val Header = FC<Props> {
+    val (theme, setTheme) = useContext(ThemeContext)
```

Comment:
I found cool hook `useTheme` - https://mui.com/styles/advanced/#accessing-the-theme-in-a-component
Seems that we can use it instead of our `ThemeModule` component :)

Reply:
Looks like there no way to set theme without the module, I see just way to access context theme by useTheme()
Do you've idea how to do it?

Reply:
Yep, seems we really can't